### PR TITLE
x updates

### PR DESCRIPTION
**Pull request by satmandu**
- Fixed for the X security issues announced today, and other updates and also added some packages needed for the incoming xterm update in progress...

Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=xupdates CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/luit.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+require 'package'
+
+class Luit < Package
+  description 'Filter that can be run between an arbitrary application and a UTF-8 terminal emulator'
+  homepage 'https://invisible-island.net/luit/luit.html'
+  version '2_0_20230201'
+  license 'custom'
+  compatibility 'all'
+  source_url 'https://github.com/ThomasDickey/luit-snapshots.git'
+  git_hashtag 'v2_0_20230201'
```
Comment (uberhacker) [inline packages/luit.rb:13]: Use `git_hashtag "v#{version}"` or `git_hashtag 'v' + version`?

Comment (satmandu) [reply packages/luit.rb:13]: The tag name varies the letters used... Otherwise, yes definitely.

File: packages/libfontenc.rb [hunk 2]
```diff
@@ -3,23 +3,23 @@
 class Libfontenc < Package
   description 'Fontenc Library from X.org'
   homepage 'https://xorg.freedesktop.org/'
-  version '1.1.6'
-  compatibility 'all'
+  version '1.1.7'
   license 'custom'
```
Comment (uberhacker) [inline packages/libfontenc.rb:7]: What does `custom` mean?

Comment (satmandu) [reply packages/libfontenc.rb]: MIT. Fixed...

File: packages/luit.rb (hunk 1 above)
Comment (uberhacker) [reply packages/luit.rb:13]: How does it vary?  Not understanding.

Comment (uberhacker) [reply packages/luit.rb:13]: Not sure what the `t` means but the `v` versions look consistent to me.

Comment (uberhacker) [reply packages/luit.rb:13]: Will we ever use a `t` release?

Comment (satmandu) [reply packages/luit.rb]: Maybe not.  Changed the file as requested.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**